python/pyspark/sql/dataframe.py

```diff
@@ -482,10 +482,22 @@ def show(self, n=20, truncate=True, vertical=False):
          age  | 5
```

Comment:
While we're here, would you mind fix the docs above `truncate : bool, optional` -> `truncate : bool or int, optional`.